packages/discovery-provider/src/tasks/metadata.py

```diff
@@ -179,7 +179,6 @@ class PlaylistMetadata(TypedDict):
     "track_cid",
     "duration",
     "is_available",
-    "is_scheduled_release",
```

Comment:
remove is_scheduled_release here to fix case where a track is uploaded as hidden, then a future release date should make it a scheduled release. removing it as immutable here so a track can go from false to true.